src/jvmTest/resources/features/screenshots.feature

```diff
@@ -34,4 +34,7 @@ Feature: Example how to describe block observations using cucumber.
       | LargeBlockArmorCornerInv      |
       | LargeHeavyBlockArmorBlock     |
       | LargeHeavyBlockArmorCornerInv |
-      | LargeHalfArmorBlock           |
\ No newline at end of file
+      | LargeHalfArmorBlock           |
+      | LargeBlockGyro                |
+      | LargeBlockSmallGenerator      |
+      | LargeBlockCockpitSeat         |
```

Comment:
Is this adding any value? Or it is just making the test run longer? It seems like it's testing the same thing whether there are 2 or 8 blocks.